test/linkedList.spec.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+const algolib = require('../index.js');
+
+describe('#linkedList', function () {
```

Comment:
You can also create a test, which returns null, when all nodes are removed from a list.